stories/QueryRuleCustomData.stories.js

```diff
@@ -0,0 +1,124 @@
+import { previewWrapper } from './utils';
+import { storiesOf } from '@storybook/vue';
+
+storiesOf('ais-query-rule-custom-data', module)
+  .addDecorator(
+    previewWrapper({
+      indexName: 'instant_search_movies',
+      filters: '<ais-refinement-list attribute="genre" />',
+      hits: `
+      <ol slot-scope="{ items }" class="playground-hits">
+        <li
+          v-for="item in items"
+          :key="item.objectID"
+          class="playground-hits-item"
+        >
+          <div
+            class="playground-hits-image"
+            :style="{ backgroundImage: 'url(' + item.image + ')' }"
+          />
+          <article>
+            <header>
+              <strong><ais-highlight attribute="title" :hit="item"/></strong>
+            </header>
+          </article>
+        </li>
+      </ol>
+      `,
+    })
+  )
+  .add('default', () => ({
+    template: `
+    <div>
+      <ul
+        <li>On query <q>music</q>: "This is it" appears.</li>
+        <li>On query <q>not a banner</q>: empty banner appears.</li>
```

Comment:
That's confusing. Do you mean no banners appear on other queries? We could remove this `li` then.

Reply:
no, there's literally an empty banner that appears in the DOM, just not visibly

Reply:
I just saw that another Query Rule was added to the `instant_search_movies` index.

![image](https://user-images.githubusercontent.com/6137112/56127058-a5937a80-5f7c-11e9-8479-d7d0376e3557.png)

We shouldn't update the index like this since it's used to demo features now.

If you want to demo something else than banners, we can for instance create an ad, which is more meaningful.

Reply:
yep, that's a good idea